contracts/mocks/DummyImplementation.sol

```diff
@@ -11,6 +14,9 @@ contract DummyImplementation {
     string public text;
     uint256[] public values;
 
+    // bytes32(uint256(keccak256('eip1967.proxy.admin')) - 1)
+    bytes32 internal constant _ADMIN_SLOT = 0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103;
```

Comment:
This can be gotten from the `ERC1967Upgrade` library once #4204 is merged

Reply:
```suggestion
```